egs/aishell2/s5/local/nnet3/tuning/run_tdnn_aishell2_ft_3a.sh

```diff
@@ -0,0 +1,66 @@
+# !/bin/bash
+
+# This script uses weight transfer as a transfer learning method to transfer
```

Comment:
I don't think the name 'aishell2' should be part of this filename.
What is the motivation for  including this script?